### ISSUE TITLE
Can't Download pretrained model

### ISSUE THREAD

**Issue by ghost** [ronghuaiyang/arcface-pytorch#42]
Hi, Baidu doesn't let me sign up for an account without a Chinese phone number. Is it possible to publish the model the other way? Thanks.

**Comment by fgvenegas** [comment]
I have the same problem

**Comment by edwardnguyen1705** [comment]
Have you guys obtained the pretrained model as well as the test dataset yet? Can anyone share on OneDrive?

**Comment by farhodfm** [comment]
Hi guys, @KaizhaoLiang, @fgvenegas, @edwardnguyen1705  

Have you obtained pretrained model and test set?
If so, could you share it, please.

Thanks in advance

**Comment by MarkZXZX** [comment]
google driver link: https://drive.google.com/drive/folders/14WeZV0o-oMax1p8soUZhezPliqUVp0gD?usp=sharing

**Comment by hanxiao120** [comment]
@MarkZXZX sorry,i can't open this page. Can you send it again?

**Comment by jonilaserson** [comment]
Hi @MarkZXZX,
Your google drive link has the `lfw-align-128.zip` file, which I thought was the one that [`lfw_test_pair.txt`](https://github.com/ronghuaiyang/arcface-pytorch/blob/master/lfw_test_pair.txt) refers to. But the `lfw_test_pair.txt` has [this line](https://github.com/ronghuaiyang/arcface-pytorch/blob/master/lfw_test_pair.txt#L934):
```
Barry_Zito/Barry_Zito_0001.jpg Barry_Zito/Barry_Zito_0002.jpg 1
```
and there is no `Barry_Zito` directory inside `lfw-align-128.zip`. How come?  Did anyone have success running `test.py`?

**Comment by sourabhyadav** [comment]
> Hi @MarkZXZX, Your google drive link has the `lfw-align-128.zip` file, which I thought was the one that [`lfw_test_pair.txt`](https://github.com/ronghuaiyang/arcface-pytorch/blob/master/lfw_test_pair.txt) refers to. But the `lfw_test_pair.txt` has [this line](https://github.com/ronghuaiyang/arcface-pytorch/blob/master/lfw_test_pair.txt#L934):
> 
> ```
> Barry_Zito/Barry_Zito_0001.jpg Barry_Zito/Barry_Zito_0002.jpg 1
> ```
> 
> and there is no `Barry_Zito` directory inside `lfw-align-128.zip`. How come? Did anyone have success running `test.py`?

Yeah, there are many such identity names whose images does not exist. II tried to remove them manually and then the test.py worked smoothly.

**Comment by harper-yuan** [comment]
> Barry_Zito

By using the publisher's link in readme(have to use a Chinese phone number to log in) to download dataset, you can find `Barry_Zito` directory! Don't use link in google drive link ,perhaps there are some problems.
